python/cuml/utils/memory_utils.py

```diff
@@ -68,7 +68,8 @@ def rmm_cupy_ary(cupy_fn, *args, **kwargs):
         temp_res = cupy_fn(*args, **kwargs)
         result = \
             _rmm_cupy6_array_like(temp_res,
-                                  order=_strides_to_order(temp_res.strides))
+                                  order=_strides_to_order(temp_res.strides,
```

Comment:
is there a test that would've caught this?

Reply:
just testing with CuPy 6.6, I think I have an idea for a follow up 0.13 tactic for some libraries like this: run the full test suite, then install the second version of the library, and run a minimal test suite (in this case `test_input_utils` is enough). Need to think about it and the consequences of it, but I think we can make it work.

Reply:
just testing with CuPy 6.6, I think I have an idea for a follow up 0.13 tactic for some libraries like this: run the full test suite, then install the second version of the library, and run a minimal test suite (in this case test_input_utils is enough). Need to think about it and the consequences of it, but I think we can make it work.